packages/lesswrong/server/voteServer.ts

```diff
@@ -253,7 +254,11 @@ export const performVoteServer = async ({ documentId, document, voteType, extend
   if (!voteTypes[voteType]) throw new Error(`Invalid vote type in performVoteServer: ${voteType}`);
 
   if (!selfVote && collectionName === "Comments" && (document as DbComment).debateResponse) {
-    throw new Error("Cannot vote on dialogue responses");
+    const post = await Posts.findOne({_id: (document as DbComment).postId});
+    const acceptedCoauthorIds = post ? getConfirmedCoauthorIds(post) : [];
```

Comment:
Note that I'm not actually sure we should be using `getConfirmedCoauthorIds`; I don't know how exactly the "pending" co-author stuff works and I'm not sure we ever used it (it might've just been the EA forum).  Is there a reason you suspect it might be necessary, rather than just checking the co-author ids on the post the way we typically do?

Reply:
Me and Ray checked now and Ray thought it seemed fine to use

Reply:
Doesn't really answer my question re: consistency, but I went and checked and it seems like the EA forum mostly uses this function and we just didn't realize it existed or something.  (I do think it's probably a bit redundant, since I don't think we actually require/check for getting permission from coauthors to be listed as coauthors anywhere, but not a big deal)

Reply:
[Speaking as Raemon on Jacob's laptop] I'm not 100% sure I understand, but my previous belief (which I still think based on how I interpret Sarah's comment) is that we basically just do use this feature on LW, we just don't think about it often). So my guess is it's just correct to use this function. Sarah, does that match your understanding?

Reply:
Yup, basically I just meant that I think it works the same on EAF and LW 👍